### PR TITLE
Remove enable sns types filter ff

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Remove `ENABLE_SNS_TYPES_FILTER` feature flag support. Now the type filtering for sns proposals is available for usage.

# Changes

- remove `ENABLE_SNS_TYPES_FILTER` feature flag support

# Tests

No changes since the flag was already enabled for unit testing.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: CHANGELOG-Nns-Dapp-unreleased.md
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp-unreleased.md:1]: Is it possible that something went wrong with this branch?

Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp-unreleased.md:1]: What do you mean w/ went wrong?

Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp-unreleased.md:1]: If you mean two records in the changelog, `* Filtering SNS Proposals by Type.` was here to emphasise that the feature is now publicly available.

File: frontend/src/lib/components/sns-proposals/SnsProposalsFilters.svelte [hunk 1]
```diff
@@ -30,17 +29,15 @@
 </script>
 
 <FiltersWrapper>
-  {#if $ENABLE_SNS_TYPES_FILTER}
```
Comment (dskloetd) [inline frontend/src/lib/components/sns-proposals/SnsProposalsFilters.svelte:33]: The normal process is to set the value of the feature flag to `true` and only some time later remove the feature flag once it's been used in production.
Is there any reason not to follow that process here?

File: CHANGELOG-Nns-Dapp-unreleased.md
Comment (dskloetd) [reply CHANGELOG-Nns-Dapp-unreleased.md:1]: When I looked, the PR had like 46 files and lots of unrelated changes. So it looked like a really bad merge.

**Comment by mstrasinskis** [comment]
Replaced w/ https://github.com/dfinity/nns-dapp/pull/4355

**Pull request closed without merging**